packages/ocamleditor/ocamleditor.1.13.4~preview.4.10/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+name: "ocamleditor"
+version: "1.1.34~preview.4.10"
+
+synopsis:
+  "OCamlEditor is a GTK+ source code editor and build tool for OCaml"
+description:
+  """It provides many features to facilitate editing code, accessing API reference
+directly from the editor and compiling projects."""
+
+authors: "Francesco Tovagliari <ftovagliari@gmail.com>"
+maintainer: "Vasile Rotaru <vrotaru.md@gmail.com>"
+
+homepage: "https://github.com/vrotaru/ocamleditor"
+bug-reports: "https://github.com/vrotaru/ocamleditor/issues"
+
+build: [["ocaml" "build.ml" "ocamleditor"]]
+install: [["ocaml" "tools/install.ml" "-prefix" prefix]]
+remove: [["ocaml" "tools/uninstall.ml" "-prefix" prefix]]
```

Comment:
```suggestion
```

This is not necessary with opam2. It keeps track of the installed files and uses the information  to uninstall the package